component/client-handler/src/main/java/org/wso2/carbon/identity/oauth2/token/handler/clientauth/jwt/validator/JWTValidator.java

```diff
@@ -392,6 +394,14 @@ private boolean logAndThrowException(String detailedMessage) throws OAuthClientA
         throw new OAuthClientAuthnException(detailedMessage, OAuth2ErrorCodes.INVALID_REQUEST);
     }
 
+    private boolean logAndThrowException(String detailedMessage, String errorCode) throws OAuthClientAuthnException {
```

Comment:
lets make the old method logAndThrowException(String detailedMessage) call this one.

Reply:
Fixed in 537c9e27acedf5c97f66c9a355f0fe58c0e52830